test/cypress/integration/consumer-tools/youth-financial-education/survey-errors.cy.js

```diff
@@ -38,7 +40,8 @@ describe( 'Youth Financial Education Survey: Errors', () => {
       .should( 'include.text', 'You forgot' );
   } );
 
-  if ( Cypress.browser.isHeaded ) {
+  // This tests fails in headless browsers, so skip it there
+  skipOn( 'headless', () => {
```

Comment:
This makes me wonder — should we even have tests that only run headed? Is that a useful thing given our reliance on headless automated test runs?

Not something that needs to be changed here, but I'm surprised to see a `Cypress.browser.isHeaded` condition at all.

Reply:
@willbarton, yeah, it is weird to have tests that won't run in any of our checks or pipelines. From what I remember, the `Cypress.browser.isHeaded` check was a workaround to get TDP tests passing in our automated runs while the TDP team worked to debug them (literal LOL: it looks like [you and I were the ones who put some of them in](https://github.com/cfpb/consumerfinance.gov/pull/6771), which I have no memory of 🤣 ). I'm fine still skipping them on headless browsers for now—at least `skipOn` reports a pending test in their place instead of ignoring them entirely like the `isHeaded` check did.

Reply:
LOL. Yeah, I think the way you've got it set up makes sense.